understanding/20/use-of-color.html

```diff
@@ -276,7 +288,7 @@ <h3>Additional Techniques (Advisory) for Use of Color</h3>
             
             <li>
                									         
-               <a href="https://www.w3.org/WAI/WCAG21/Techniques/css/C15" class="css"></a>
+               <a href="https://www.w3.org/WAI/WCAG21/Techniques/css/C15" class="css">Using CSS to change the presentation of a user interface component when it receives focus</a>
```

Comment:
@patrickhlauke the generation process should take care of this - we often leave the title out so if it changes it isn't incorrect (but the generator should replace the anchor content anyway).

Reply:
ah, good to know. would you prefer me to remove the text, for consistency/style's sake?

Reply:
noting that all other links to techniques in this understanding doc have explicit link text, so perhaps for consistency within this document itself probably worth keeping (or changing the other ones to be empty)